From 9df309bde02edf7017cf3cad4eaeb2f22404cf6b Mon Sep 17 00:00:00 2001
From: Kunhui-Li <kunhuix.li@intel.com>
Date: Wed, 18 Aug 2021 13:45:52 +0800
Subject: [PATCH 13/13] config_tools: disable TPM2 passthrough on other
 platforms

Disables TPM2 passthrough on adl-s-crb platform follow the 6410 PR.

Tracked-On: #6288
Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>
---
 misc/config_tools/data/adl-s-crb/hybrid_rt.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/config_tools/data/adl-s-crb/hybrid_rt.xml b/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
index 3f94d6581..60a5a120a 100644
--- a/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
+++ b/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
@@ -121,7 +121,7 @@
 	    <pci_dev>00:17.0 SATA controller: Intel Corporation Device 7ae2 (rev 10)</pci_dev>
     </pci_devs>
     <mmio_resources>
-      <TPM2>y</TPM2>
+      <TPM2>n</TPM2>
     </mmio_resources>
   </vm>
   <vm id="1">
-- 
2.25.1

